mavros_msgs/msg/HilStateQuaternion.msg

```diff
@@ -0,0 +1,12 @@
+# HilStateQuaternion.msg
+#
+# ROS representation of MAVLink HIL_STATE_QUATERNION
+# See mavlink message documentation here:
+# https://pixhawk.ethz.ch/mavlink/#HIL_STATE_QUATERNION
+
+std_msgs/Header header
+sensor_msgs/Imu imu
+geometry_msgs/Vector3 linear_velocity
+sensor_msgs/NavSatFix fix
```

Comment:
Same idea about NavSatFix, but Imu may be ok (except that i don't like waste of header and variances).